.github/workflows/ci-pr.yml

```diff
@@ -128,6 +128,10 @@ jobs:
             - name: 'Run Apex tests'
               run: sfdx force:apex:test:run -c -r human -d ./tests/apex -w 20
 
+            # Run Jest tests
+            - name: 'Run Jest tests'
```

Comment:
I'm surprised to see that we weren't already running Apex tests as part of CI, my bad for not checking this earlier.

However, we don't want to do it this way, there are a number of things that we need to fix and I suggest a separate PR for this.

We need to be consistent with the other CI workflows [like this one](https://github.com/trailheadapps/lwc-recipes/blob/main/.github/workflows/ci-pr.yml#L62):
- run LWC tests on all 3 CI workflows, not just `ci-pr`
- use the `@salesforce/sfdx-lwc-jest` node library directly instead of the CLI so that we control the version
- run tests in the `format-lint-lwc-tests` job, not `scratch-org-test`
- run the Codecov action with the LWC flag just after

Reply:
I'll revert that change momentarily.